### PR TITLE
Updated the list of EIP Editors

### PR DESCRIPTION
**Pull request by yashkamalchaturvedi**
I have updated the list of old EIP editors with the new one.

### CODE REVIEW

**Review by lightclient — APPROVED** [review]

File: src/content/eips/index.md [hunk 1]
```diff
@@ -25,16 +25,22 @@ Along with providing a technical specification for changes, EIPs are the unit ar
...
+EIP editors are tasked with process of reviewing EIPs for technical soundness, correct spelling/grammar, and code style. Martin Becze, Vitalik Buterin, Gavin Wood, and a few others were the original EIP editors from 2015 to late 2016. 
+
+The current EIP editors are:
+- Alex Beregszaszi (@axic)
+- Matt Garnett (@lightclient)
+- Micah Zoltu (@MicahZoltu)
+- Greg Colvin (@gcolvin)
+- Sam Wilson (@SamWilsn)
+
+Emeritus EIP editors are:
```
Comment (minimalsm) [inline src/content/eips/index.md:37]: We translate this page, let's keep the language simple and understandable here.

```suggestion
Retired EIP editors are:

```

File: src/content/eips/index.md [hunk 2]
```diff
@@ -25,16 +25,23 @@ Along with providing a technical specification for changes, EIPs are the unit ar
...
-- Greg Colvin (Community)
-- Casey Detrio (EWASM/Ethereum Foundation)
-- Matt Garnett (Quilt)
-- Hudson James (Ethereum Foundation)
-- Nick Johnson (ENS)
-- Nick Savers (Community)
-- Micah Zoltu (Community)
+EIP editors are tasked with process of reviewing EIPs for technical soundness, correct spelling/grammar, and code style. Martin Becze, Vitalik Buterin, Gavin Wood, and a few others were the original EIP editors from 2015 to late 2016. 
+
+The current EIP editors are:
```
Comment (minimalsm) [inline src/content/eips/index.md:30]:
```suggestion
The current EIP editors are:

```

**Review by minimalsm — APPROVED** [review]
Thanks for the PR @yashkamalchaturvedi ❤️  merging now.

And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @yashkamalchaturvedi for docs

**Pull request merged**